stat11_backend/stat11/utils/match_team_list_data.py

```diff
@@ -0,0 +1,45 @@
+from stat11.models import Team, Player
+from stat11.serializers import TeamSerializer , PlayerNestedSerializer
+
+def get_match_team_list_data(match_id):
+    teamList = []
+    matchTeams = Team.objects.filter(match__id = match_id)
+    firstTeam = matchTeams[0]
+    secondTeam = matchTeams[1]
+    serializer1 = TeamSerializer(firstTeam)
+    data1 = serializer1.data
+    serializer2 = TeamSerializer(secondTeam)
+    data2 = serializer2.data
+    teamList.append(data1['players'])
+    teamList.append(data2['players'])
+    netTeam = []
+    for i,pl in enumerate(teamList):
+        team = []
+        for j,pid in enumerate(pl):
+            # print(pid)
+            player = Player.objects.all()
+            for k in player:
+                if(k.id == pid):
+                    data = k
+            
+            if(data.type == 'all_rounder'):
+                Type = 'All Rounder'
+            elif(data.type == 'bowler'):
+                Type = 'Bowler'
+            else:
+                Type = 'Batter'
+            
```

Comment:
You don't need the above code block......We have already catered the above thing while defining the choices in the constants.....Look at it once

Reply:
Will need to change some code in frontend too, if it can be kept then let it be

Reply:
But send the data....it will appear like this only.....you don't need these if-else blocks.....just read the code for once and send the fields directly

Reply:
Removed the If else code and sent it directly now, will look into the frontend changes later